nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/PutAzureBlobStorage.java

```diff
@@ -20,9 +20,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URISyntaxException;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Minor thing: I'd suggest not using a wildcard import here.

Reply:
The IDE did automatic imports, I have not fixed them

Reply:
The IDE did automatic imports, I have now fixed them